src/index.js

```diff
@@ -1,13 +1,14 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import 'bootstrap/dist/css/bootstrap.min.css';
+import { BrowserRouter } from 'react-router-dom';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
 
 ReactDOM.render(
-  <React.StrictMode>
+  <BrowserRouter>
     <App />
```

Comment:
- [ ] The layout of your pages does not conform to the layout provided in the given [wireframe](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/images/math_magicians_home.png) template. We actually want our design to as much as possible to follow the [wireframe](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/images/math_magicians_home.png) template. Kindly ensure to improve upon this aspect, ensuring each page fills the whole screen, as can be seen in the template design.

**Your design:**
![image](https://user-images.githubusercontent.com/12745474/145636791-e1894ace-339d-46a6-b761-eb23b6c3d498.png)


![image](https://user-images.githubusercontent.com/12745474/145636700-37104653-1b1e-47fc-b64e-a7189e324ced.png)

![image](https://user-images.githubusercontent.com/12745474/145636634-a9638d37-8deb-49ce-8b22-f85c3dfac77d.png)

**The wireframe design:**
![image](https://user-images.githubusercontent.com/12745474/145637172-00782574-7298-4f40-9c67-984243514a55.png)